### PR TITLE
Create issue.yaml

### PR DESCRIPTION
**Pull request by Erin-Cecele**
This is still in progress.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+labels: [feature]
+assignees:
+- erin-cecele
+- geealbers
+body:
+- type: markdown
+  attributes:
+    value: |
+   
+      **Thank you for opening a new issue!**
```
Comment (geealbers) [inline .github/ISSUE_TEMPLATE/issue.yaml:12]: Did you want to make the same heading style change here as you did in the other repo?

**Review by geealbers — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/issue.yaml (hunk 1 above)
Comment (quire-user) [reply .github/ISSUE_TEMPLATE/issue.yaml:12]: I just did this but I accidentally put it in PR #430 Create pull request template for docs. 

If I merge this one first and then 430 I shouldn't have any conflicts, correct?

Comment (geealbers) [reply .github/ISSUE_TEMPLATE/issue.yaml:12]: Ah, I see. Yes, that should work. I think. :)

Comment (Erin-Cecele) [reply .github/ISSUE_TEMPLATE/issue.yaml:12]: Oops, accidentally commented as @quire-user.

**Pull request merged**